mipengine/node/tasks/remote_tables.py

```diff
@@ -5,7 +5,6 @@
 from mipengine.common.node_tasks_DTOs import TableInfo
 from mipengine.node.monetdb_interface import remote_tables
 
-
 # TODO: the database name has to come from somwhere outside this module. Not from the controller, controller should not know that
 DB_NAME = "db"
```

Comment:
DB_NAME is not used, remove it?
Also the caller of the task/function `create_remote_table` should pass the `table_info` and the ip and port of the monetdb where the "native" table exists.
It should not pass the `api:monetdb://` prefix, since the caller should not know about the protocol used to make the communication between the to monetdbs, it just says here is the address of the machine. It should also not pass the database name. The database name is internal information of the node, the caller must not know which db the node is talking to.

Reply:
This will be fixed on the [MIP-16](https://team-1617704806227.atlassian.net/jira/software/projects/MIP/boards/1?selectedIssue=MIP-16) .